ga4gh/backend.py

```diff
@@ -90,19 +91,18 @@ def runSearchRequest(
             request.pageSize = self._defaultPageSize
         if request.pageSize <= 0:
             raise exceptions.BadPageSizeException(request.pageSize)
-        pageList = []
+        response = protocol.SearchResponseBuffer(
```

Comment:
Interesting. So `response` is no longer explicitly an instance of `responseClass` as best as I can tell, even though it passes validation against the `responseClass` later on (with the caveat of converting the string to a dict via `json.loads`).